### PR TITLE
Update README.md and level.json

### PR DESCRIPTION
**Pull request by ghost**
* Updated README.md to include game information.
* Update number of cats in level.json.
* Increase max puzzle count for extended gameplay and even to reach final level.

Kindly Review @walterbender @chimosky @quozl .

### CODE REVIEW

**Comment by ghost** [comment]
Though my initial plan was to update the level and its type in level.json itself but it turns out some of the attributes are hardcoded so just updated number of cats only.
Thanks.

**Comment by ghost** [comment]
Added Win Message on flawlessly completing all levels and increased max_attempts = 10 ,which is equal to max level that can be reached ,so there is one way to win and that is without losing once.
Before it was max_attempts = 5, due to which only 5 levels could be reached, which doesnt make any sense so corrected it.
Also let say if a player loses once he will continue to play but he can max reach level 9 and if loses 2 max he can reach is 8 and so on which is how game was designed continued with that as it keeps you in game until you win all the way.

File: README.md [hunk 1]
```diff
@@ -1 +1,22 @@
-# clasiffy-cats
\ No newline at end of file
+# Classify-Cats
+## A sugar activity where players classify cats based on various criteria.
```
Comment (chimosky) [inline README.md]: Shouldn't be a header.

**Comment by chimosky** [comment]
Your commit messages tell us about the change you've made but not why, and it'll be great to know why these changes were made - particularly the changes to `level.json`.

File: README.md (hunk 1 above)
Comment (ghost) [reply README.md]: Done!

**Comment by ghost** [comment]
Kindly see the commit messages now if it provides more context.

**Comment by ghost** [comment]
These changes seems to be dependent with other PR standalone they aren't any good, which is a grave mistake done by me, will not repeat it again, will try to generate one pr only, thanks.

**Pull request closed without merging**